internal/app/api/api.go

```diff
@@ -314,6 +314,9 @@ func (a *Client) GetFeed(ctx context.Context, request *GetFeedRequest) (*GetFeed
 	if err != nil {
 		return nil, errors.Wrap(err, "Failed request to API")
 	}
+	if errMsg != nil && len(errMsg) > 0 {
+		return nil, fmt.Errorf(string(errMsg))
```

Comment:
Do you have an example of what this looks like?

Reply:
Failed to export feed: {"statusCode":403,"error":"Forbidden","message":"The caller does not have permission to execute the specified operation"}